src/elm/Help/Help.elm

```diff
@@ -212,7 +215,11 @@ docsLink : Command -> Html msg
 docsLink command =
     case command.docs of
         Just docs ->
-            a [ class "cmd-link", href <| cliDocsUrl docs ]
+            a
+                [ class "cmd-link"
+                , href <| cliDocsUrl docs
+                , attribute "aria-label" <| "go to cli docs page for " ++ docs
```

Comment:
adding better aria label for docs link.
before, screen reader just read "link, docs"